src/results/ResponseMessageResult.ts

```diff
@@ -0,0 +1,12 @@
+import { HttpResponseMessage } from "../httpResponseMessage";
+import { CREATED } from "http-status-codes";
```

Comment:
Well this is wrong.  I obviously missed this when I was making some templates, but I'm pretty surprised linting didn't catch this.

Want me to look into fixing the lint rules, or just fix the imports?